src/ext/FirewallExtension/ca/firewall.cpp

```diff
@@ -39,7 +39,7 @@ static UINT SchedFirewallExceptions(
     LPWSTR pwzCustomActionData = NULL;
     LPWSTR pwzName = NULL;
     LPWSTR pwzRemoteAddresses = NULL;
-    int iPort = 0;
+    LPWSTR pwzPort = NULL;
```

Comment:
Needs a matching call to ReleaseStr at LExit.

Have you signed an assignment agreement? We'll need that. See http://wixtoolset.org/development/assignment-agreement/ for details.